src/Components/Header.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import teamlogo from "../asset/teamlogo.jpg";
+import user from "../asset/user.png";
+
+const HeaderComponent = () => {
+  return (
+    <div className="header">
+      <Logo></Logo>
+      <SearchBar></SearchBar>
```

Comment:
SearchBar is neither imported nor present in this file. Is this not throwing error. or Am i missing anything here.

Reply:
Actually it was for 4th assignment moved the header inside body now. Forgot to remove it